packages/api/src/codegen/languages/typescript/index.ts

```diff
@@ -95,10 +96,22 @@ export default class TSGenerator extends CodeGenerator {
             : corePkg.version,
       },
       'json-schema-to-ts': {
+        dependencyType: 'production',
         reason: 'Required for TypeScript type handling.',
         url: 'https://npm.im/json-schema-to-ts',
         version: '^2.9.2',
       },
+      tsup: {
+        dependencyType: 'development',
+        reason: "Used for compiling your codegen'd SDK into code that can be used in JS environments.",
+        url: 'https://tsup.egoist.dev/',
+        version: '^7.2.0',
+      },
+      typescript: {
+        dependencyType: 'development',
+        reason: 'Required for `tsup`.',
+        version: '^5.2.2',
```

Comment:
if and when we improve the codegen for our test fixtures (either via a build script or something), it'd be cool if we can dynamically pull in the versions from our dependencies

Reply:
100%